api/src/main/java/org/apache/cloudstack/api/command/admin/user/UpdateUserCmd.java

```diff
@@ -80,6 +80,9 @@ public class UpdateUserCmd extends BaseCmd {
     @Parameter(name = ApiConstants.USERNAME, type = CommandType.STRING, description = "Unique username")
     private String username;
 
+    @Parameter(name = ApiConstants.MANDATE_2FA, type = CommandType.BOOLEAN, description = "Provide true to mandate the user to use two factor authentication has to be enabled. This parameter is only used to mandate 2FA, not to disable 2FA")
```

Comment:
@harikrishna-patnala 
is `mandate2fa`  a parameter only for admins (root admin, domain admin) ?

Reply:
Yes, thanks @weizhouapache updated the parameter.